setup/options.rst

```diff
@@ -0,0 +1,228 @@
+Options
+=======
+
+There are a number of options for OpenRCT2 that can be tuned to fit your preferences and needs. They are broken up into a number of different sections.
+
+Display
+-------
+
+There are a number of options dedicated to how the game runs on your screen.
+
+* **Screen mode**
+
+  Can be one of three values:
+
+  * *Windowed* - the game will run inside a window with standard decorations like minimize and close buttons.
```

Comment:
We use `-ise` suffixes - so `minimise` rather than `minimize`. There are three occurrences of -ize in this document.